src/core_write.cpp

```diff
@@ -129,14 +129,14 @@ std::string EncodeHexTx(const CTransaction& tx, const int serializeFlags)
 }
 
 void ScriptPubKeyToUniv(const CScript& scriptPubKey,
-                        UniValue& out, bool fIncludeHex)
+                        UniValue& out, bool include_hex)
 {
     txnouttype type;
     std::vector<CTxDestination> addresses;
     int nRequired;
 
     out.pushKV("asm", ScriptToAsmStr(scriptPubKey));
-    if (fIncludeHex)
```

Comment:
Nit, since you are updating style here, add `{ }`?

Reply:
fixed in the second commit

Reply:
Missing here.